tests/test_download.py

```diff
@@ -16,70 +16,66 @@
 import os
 import shapely
 from pathlib import Path
-from osm_flex.download import (_create_gf_download_url, 
-                               get_country_geofabrik, 
-                               get_region_geofabrik, 
-                               get_planet_file)
+from osm_flex.download import (
+    _create_gf_download_url,
+    get_country_geofabrik,
+    get_region_geofabrik,
+    get_planet_file,
+)
 
-class TestDownload(unittest.TestCase):
 
-    def setUp(self):
-        pass
+class TestDownload(unittest.TestCase):
 
     def test_create_gf_download_url(self):
-        
-        # TODO: assert raises KeyError
-        #iso3 = 'ATG'
-        #file_format = 'pbf'
-        #_create_gf_download_url(iso3, file_format)
-        
-        # TODO: assert raises KeyError
-        #iso3 = 'DEU'
-        #file_format = 'pdf'
-        #_create_gf_download_url(iso3, file_format)
-        
-        
-        iso3 = 'DEU'
-        file_format = 'pbf'
-        url = _create_gf_download_url(iso3, file_format)
-        self.assertEqual(url, 'https://download.geofabrik.de/europe/germany-latest.osm.pbf')
-        
-        iso3 = 'DEU'
-        file_format = 'shp'
-        url = _create_gf_download_url(iso3, file_format)
-        self.assertEqual(url, 'https://download.geofabrik.de/europe/germany-latest-free.shp.zip')
-        
-        iso3 = 'RUS-A'
-        file_format = 'pbf'
-        url = _create_gf_download_url(iso3, file_format)
-        self.assertEqual(url, 'https://download.geofabrik.de/asia/russia-latest.osm.pbf')
-        
-        iso3 = 'RUS-E'
-        file_format = 'pbf'
-        url = _create_gf_download_url(iso3, file_format)
-        self.assertEqual(url, 'https://download.geofabrik.de/europe/russia-latest.osm.pbf')
-    
-        
+        with self.assertRaisesRegex(
+            KeyError, "The provided iso3 seems not to be available on Geofabrik.de"
+        ):
+            _create_gf_download_url("ATG", "pbf")
```

Comment:
```suggestion
            _create_gf_download_url("XXX", "pbf")
```
I would use something that does not exist at all. Otherwise, if geofabrik ads the data for `ATG` in the future, this test would fail.

Reply:
True!